### PR TITLE
Clarify exotic hardware requirements

### PR DESCRIPTION
**Pull request by cpaelzer**
Be less misunderstandbale (we mean the HW not the PKG) and provide a few examples so requestors know what an answer might look like.

Fixes: #61

### CODE REVIEW

**Review by didrocks — APPROVED** [review]
This is way clearer to me, thanks for drafting this!

File: README.md [hunk 1]
```diff
@@ -334,8 +334,18 @@ TODO:   - Debian https://bugs.debian.org/cgi-bin/pkgreport.cgi?src=TBDSRC
 TODO:   - Upstream's bug tracker, e.g., GitHub Issues
 TODO: - The package has important open bugs, listing them: TBD
 TODO-A: - The package does not deal with exotic hardware we cannot support
-TODO-B: - The package does deal with exotic hardware, it is present at TBD
-TODO-B:   to be able to test, fix and verify bugs
+TODO-B: - The package does deal with exotic hardware, such hardware is available
+TODO-B:   to the team for debugging, test, verification and development via
```
Comment (setharnold) [inline README.md:338]: Nitpick, a `:` at the end would be nice.

**Review by setharnold — APPROVED** [review]
One small stylistic suggestion, a colon.

File: README.md (hunk 1 above)
Comment (cpaelzer) [reply README.md]: discussed and resolved in a follow on commit

**Review by slyon — APPROVED** [review]
lgtm

**Comment by slyon** [comment]
This is looking all good. Let's get it merged!

**Pull request merged**